inductiva/molecules/scenarios/md_water_box/md_water_box.py

```diff
@@ -23,19 +23,14 @@ class MDWaterBox(Scenario):
     def __init__(
         self,
         temperature: float = 300,
-        box_size: float = 3.0,
+        box_size: float = 2.3,
```

Comment:
Why this change? :)

Reply:
Let's inform the user about that!

Reply:
Updated documentation!

Reply:
Ok, what I meant was to maybe add a logging.info telling the user what happened, instead of him requiring to go see the docs about why the simulation failed.

Reply:
Ah, I've corrected it :)